src/main/scala/piconot/external/Empty.bot

```diff
@@ -0,0 +1,12 @@
+    state(left)(
```

Comment:
Looking at this, I like most of it, but it's a little unclear to me what to(fill) means on first glance.